package.json

```diff
@@ -72,6 +72,7 @@
         "socks-proxy-agent": "^8.0.4",
         "stream-buffers": "^3.0.2",
         "tar": "^7.0.0",
```

Comment:
`tar` should be removed from this file and `package-lock.json`

Reply:
fixed.